contrib/mesos/pkg/executor/service/service.go

```diff
@@ -173,6 +173,19 @@ func (s *KubeletExecutorServer) runKubelet(
 		kcfg.DockerDaemonContainer = "" // don't move the docker daemon into a cgroup
 		kcfg.Hostname = kcfg.HostnameOverride
 		kcfg.KubeClient = apiclient
+
+		// taken from KubeletServer#Run(*KubeletConfig)
+		clientConfig, err := s.CreateAPIServerClientConfig()
+		if err == nil {
+			kcfg.KubeClient, err = client.New(clientConfig)
```

Comment:
I literally copied this code 1:1 from upstream as suggested by @sttts, we had `apiclient` in the first place, but apiclient does not have those QPS and Burst settings.

Shall I still use apiclient?